server/tests/constants.js

```diff
@@ -0,0 +1,19 @@
+export const TEST_ACCESS_TOKEN = '123'
```

Comment:
@AStashevskaya You can use `nconf` module in combination with `dotenv` to set default variables for environment variables instead